patterns/multisig/proxy.ts

```diff
@@ -0,0 +1,136 @@
+import { types } from "polkadot_dev/mod.ts"
+import { Proxy, Utility } from "polkadot_dev/mod.ts"
+import { Event } from "polkadot_dev/types/pallet_proxy/pallet.ts"
+import { RuntimeEvent } from "polkadot_dev/types/polkadot_runtime.ts"
+import { ArrayRune, MultiAddress, Rune, RunicArgs } from "../../mod.ts"
+import { hex, Sr25519 } from "../../util/mod.ts"
+
+export interface BuildPureProxyMultisigProps {
+  sender: Sr25519
+  admins: Uint8Array[]
+  threshold?: number
+}
+
+export function buildPureProxyMultisig<X1>(props: RunicArgs<X1, BuildPureProxyMultisigProps>) {
```

Comment:
How do we feel about the name `MultiProxy`?

Reply:
it should be as idiomatic as possible IMO. I should be able to look at the official polkadot/substrate and easily connect it to whats in capi. PureProxyMultisig makes it obvious what it does.